fix: add deprecated badge to nav (#1569)
contentful/forma-36: packages/forma-36-website/src/content/deprecated-components/portal/index.mdx

---
title: 'Portal'
status: 'deprecated'
---

# Portal

> Portal component got removed in version 4. In React, Portal is used to render component or element outside the DOM hierarchy of the parent component.
> 
> If you need to use Portal for any situation, have a look at the [React documentation around this topic](https://reactjs.org/docs/portals.html)

Reviewer: I would remove that explanation about what React Portal is. Since there is a great description in the official React docs (that you are pointing to below).
```suggestion
The Portal component got removed in version 4.

If you need to use Portal for any situation, have a look at the [official React documentation](https://reactjs.org/docs/portals.html)
```
